tests/adapter/dbt/tests/adapter/utils/test_timestamps.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from dbt.tests.util import check_relation_has_expected_schema, run_dbt
+
+_MODEL_CURRENT_TIMESTAMP = """
+select {{ current_timestamp() }} as current_timestamp,
+       {{ current_timestamp_in_utc_backcompat() }} as current_timestamp_in_utc_backcompat,
+       {{ current_timestamp_backcompat() }} as current_timestamp_backcompat
+"""
+
+
+class TestCurrentTimestamps:
```

Comment:
The patten I'm used to seeing is like [this](https://github.com/dbt-labs/dbt-core/blob/c2ab2971b073245366f1ccd8a9e1cbe198f18865/tests/adapter/dbt/tests/adapter/grants/test_invalid_grants.py).

```
import pytest
from dbt.tests.adapter.category.base_thing import BaseThing

class BaseSpecificThing(BaseThing):
    <a bunch of code>

class TestSpecificThing(BaseSpecificThing):
    pass
```

---

Is there a purpose for separating these two files that I'm overlooking?
- `tests/adapter/dbt/tests/adapter/utils/test_timestamps.py`
- `tests/functional/timestamps/test_timestamps.py` 

Or could they be a single file like the pattern above?

---

Assuming they can be a single file, then a three-part suggestion:
1. Make the update suggested below
1. Add the trivial wrapper to the end of this file
1. Remove the other wrapper file

### Implementation details
1. Suggested update
```suggestion
class BaseCurrentTimestamps:
```

2. Add this at the end of the file:
```
class TestCurrentTimestamps(BaseCurrentTimestamps):
    pass
```

3. If you do this, then you can drop the `tests/functional/timestamps/test_timestamps.py` file altogether.